### PR TITLE
Update RepeatMasker to 4.1.5

### PR DESCRIPTION
**Pull request by abretaud**
Update RepeatMasker to 4.1.5

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

**Comment by abretaud** [comment]
Arf, "no space left"... RepeatMasker comes with a 5.5Gb Dfam.h5 file. Checking if maybe it could be downloaded from https://www.dfam.org/releases/Dfam_3.7/families/ with a post-link script :thinking:

**Comment by bgruening** [comment]
It that new? post-link will prevent to have a 5.5 GB tarball at anaconda.org but it still will fetch the data here and fail. Only thing we can do is to build it locally and push I think.

**Comment by abretaud** [comment]
Ah yep, I see. It uses this file (same sha256, checked): https://www.dfam.org/releases/Dfam_3.7/families/Dfam_curatedonly.h5.gz
=> 5.5Gb unzipped, but 836Mb in the archive

I guess the file was just a little smaller in previous version.

If you can build and manually push it's great, otherwise the only way would be to remove this file and let users download it manually? Which is a bit painful for galaxy

**Comment by bgruening** [comment]
The package is already 200MB: https://anaconda.org/bioconda/repeatmasker/files

And now its getting much bigger? I think then a post-link script is needed. I than build it locally and push - which will lead to a big container but small conda tarball on anaconda.com

File: recipes/repeatmasker/build.sh [hunk 1]
```diff
@@ -5,10 +5,17 @@ mkdir -p ${PREFIX}/bin
 mkdir -p ${RM_DIR}
 cp -r * ${RM_DIR}
 
+export LANGUAGE=en_US.UTF-8
```
Comment (bgruening) [inline recipes/repeatmasker/build.sh:8]: Why is this needed?

Comment (abretaud) [reply recipes/repeatmasker/build.sh:8]: There were some perl warning about that in previous build, they're gone with that change

**Comment by abretaud** [comment]
It makes sense in fact: my latest commit removes 5,5Gb from the disk + a 1Gb zipped copy in the generated package, which is enough to finish the rest of the build. I think it's ok like this

**Comment by abretaud** [comment]
@BiocondaBot please add label

**Review by bgruening — APPROVED** [review]
